core/services/feeds/orm.go

```diff
@@ -103,3 +106,79 @@ func (o *orm) CountManagers() (int64, error) {
 
 	return count, nil
 }
+
+// CreateJobProposal creates a job proposal.
+func (o *orm) CreateJobProposal(ctx context.Context, jp *JobProposal) (uint, error) {
```

Comment:
Since `id` is an int64 in postgres internally, it might be nice to be consistent and use an `int64` in go as well